features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml

```diff
@@ -29,7 +29,6 @@
       <import feature="org.eclipse.wst.web_ui.feature" version="3.7.1.v201602111638" match="greaterOrEqual"/>
    </requires>
 
-   <!-- appengine-plugins-core -->
```

Comment:
This comment is a pain in the neck as any change using the feature editor results in its being thrown away.

Reply:
I wondered why that happened. Let's just let it go.